apps/firebase-api/src/app/users.ts

```diff
@@ -0,0 +1,56 @@
+import admin from 'firebase-admin';
+import { logger } from 'firebase-functions';
+
+const getUserRef = (userId) =>
+  admin.firestore().collection('users').doc(userId);
+
+const addUNumToAllUsers = () => {
+  let uNum = 1;
+
+  const iterateAllUsers = async (nextPageToken?) => {
+    // List batch of users, 1000 at a time.
+    try {
+      const listUsersResult = await admin.auth().listUsers(1, nextPageToken);
+      const batch = admin.firestore().batch();
+
+      listUsersResult.users.forEach((userRecord) => {
+        uNum++;
```

Comment:
This should come after the next statement. Otherwise, your first user will be a `uNum` of 2.